cfgen/cfgen

```diff
@@ -1513,6 +1563,93 @@ class ConfProfile(ToDhall):
         return prof_id
 
 
+class ConfFdmiFltGrp(ToDhall):
+    _objt = ObjT.fdmi_flt_grp
+    _downlinks: Set[Downlink] = set()
+
+    def __init__(self, rec_type: int, fdmi_filters: List[Oid]):
+        assert all(x.type is ObjT.fdmi_filter for x in fdmi_filters)
+        self.rec_type = rec_type
+        self.fdmi_filters = fdmi_filters
+
+    def __repr__(self):
+        args = ', '.join([f'rec_type={self.rec_type}',
+                          f'fdmi_filters={self.fdmi_filters!r}'])
+        return f'{self.__class__.__name__}({args})'
+
+    def to_dhall(self, oid: Oid) -> str:
+        assert oid.type is ObjT.fdmi_flt_grp
+        args = ', '.join([f'id = {oid_to_dhall(oid)}',
+                          f'rec_type = {self.rec_type}',
+                          f'filters = {oids_to_dhall(self.fdmi_filters)}'])
+        return '{ %s }' % args
```

Comment:
```suggestion
        return f'{ {args} }'
```

Reply:
I did exactly as it was done in other the places, to keep the style uniform. I will make the suggested change though. Please let me know if it would be better to still keep the style uniform.